### ISSUE TITLE
Incompatibility with version 7.01 (beta 4425)

### ISSUE THREAD

**Issue by grimsi** [shayne/hwinfo-streamdeck#28]
It seems like the plugin does not work with the latest beta of 7.01 (it shows "Please launch HWiNFO64" even if the tool is running and shared memory is enabled).
This is *not* a duplicate of #27 since it is not working at all, even with the Pro version.

Maybe the access to the shared memory changed in 7.01?

**Comment by grimsi** [comment]
Also I wanted to add that the "persistent driver" functionality is enabled as advised [here](https://github.com/shayne/hwinfo-streamdeck/issues/22#issuecomment-705460153)

**Comment by bezibaerchen** [comment]
+1
Am on HiNFO64 Pro Version 7.05-4485
Streamdeck states to please launch HWiNFO64 hence shared memory support enabled.

**Comment by EpicLPer** [comment]
I found a Q&A from their forum and sadly, as it seems, they now limit API usage to 12 hours or so for the free version.

```
Q: What plugins and extensions will be affected by the SHM limitation?
A: All that make use of the SHM interface to pull data from HWiNFO64: HWiNFOMonitor, Rainmeter plug-in, Prometheus Adapter, Aquacomputer Aquasuite, MSI Afterburner via the HWiNFO plugin, Elgato Streamdeck, TThrottle, etc. RTSS support built-in HWiNFO will not be affected.

Q: What were the reasons behind limiting the SHM also for personal users?
A: Current requirement for allowing other applications to use SHM is that the product must be an unconditional freeware while keeping the interface details closed. The main reason behind this is not to allow commercial products to make use of (and earn from) a freeware product. However this was difficult to meet for many projects - open-source projects to keep a part of code closed, or projects that were somehow related to commercial products or were earning even indirectly. Moreover, some subjects have attempted to circumvent the SHM restriction and use it for commercial purposes by utilizing illegal methods. So the new SHM model is uniform without implementation restrictions and allows full disclosure of the interface. I also believe that a 12-hour limit on SHM is sufficient for most personal users and will deter some of the commercial usage.
```

**Comment by shayne** [comment]
Fixed in v2.0.0

**Comment by FormBurden** [comment]
Thanks for this quick fix, really appreciate it!
